.devcontainer/.gitignore

```diff
@@ -0,0 +1 @@
+.zsh_history
```

Comment:
@sindhupavanimovva maybe it's time to set up some basic pre-commit checks - one rule in git is that the final line in a text file is always present but blank (it helps make diffs at the end of files clearer), so adding some checks would prevent issues like this.

Also, you could check that your IDE isn't removing that last blank line on save.